puppet/spec/unit/indirector/catalog/puppetdb_spec.rb

```diff
@@ -50,7 +51,7 @@ def save
     end
 
     it "should log a deprecation warning if one is returned from PuppetDB" do
-      response['x-deprecation'] = 'A horrible deprecation warning!'
+      response.nethttp['x-deprecation'] = 'A horrible deprecation warning!'
```

Comment:
you should be able to access the response headers through the `response[]` directly